src/scripts/cronjobs/firstDataBrokerRemovalFixed.tsx

```diff
@@ -19,24 +19,25 @@ import { sanitizeSubscriberRow } from "../../app/functions/server/sanitize";
 import { refreshStoredScanResults } from "../../app/functions/server/refreshStoredScanResults";
 import { getScanResultsWithBroker } from "../../db/tables/onerep_scans";
 import { hasPremium } from "../../app/functions/universal/user";
+import { logger } from "../../app/functions/server/logging";
 
 type SubscriberFirstRemovedScanResult = {
   subscriber: SubscriberRow;
   firstRemovedScanResult: OnerepScanResultDataBrokerRow;
 };
 
-function isFulfilledResult(
-  result: PromiseSettledResult<SubscriberFirstRemovedScanResult | undefined>,
-): result is PromiseFulfilledResult<SubscriberFirstRemovedScanResult> {
-  return (
-    typeof result !== "undefined" &&
-    result.status === "fulfilled" &&
-    typeof result.value !== "undefined"
-  );
-}
+process.on("SIGINT", () => {
+  logger.info("SIGINT received, exiting...");
+  tearDown();
+});
 
 void run();
 
+function tearDown() {
```

Comment:
nit: the teardown can be pulled into a shared function.

Reply:
You mean across the different cronjobs? I think what specifically needs to be teared down differs per cron job, i.e. some will need to close down the email pool, but others will never send email. So then it's probably best to keep the cron job in control?